### ISSUE TITLE
How To Set AutoPlaySetTimeOut for each item?

### ISSUE THREAD

**Issue by kanialda** [OwlCarousel2/OwlCarousel2#1996]
Dear OwlCarousel2 Team, 

I would like to ask about timeout in owlcarousel. Is it possible to set timeout for each item in carousel?
Because my project needs to implement different timeout in each image.

Thanks for your advice.

**Comment by keefyhub** [comment]
I dont think it does anything like this out of the box, you could write some custom functionality to trigger a "next.owl.carousel" event.

Something like this - https://codepen.io/keefyboooo/pen/OjRaZg

So after the plugin has initialized, I get the current active item and check for a `data-timing` attribute, if one doesnt exist then I default back to 5000 (5 seconds).

Then I added the `translated.owl.carousel` so that after the first slide has been transitioned by the initialize event, it will continue to run.

Seems to work with or without loop.

Hope that helps!
